app/models/project_accumulator.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+class ProjectAccumulator
+  def initialize(project:, session_id:)
+    @collection_id = project.mediaflux_id
+    @session_id = session_id
+  end
+
+  # Create accumulators for all newly created mediaflux projects
+  #
+  def create!
+    accum_names = validate
+    return true if accum_names == true
```

Comment:
What does the true signify here?  I might have expected true to signify a change and false to signify no change.  What does the method return of a change has occurred?

Reply:
@carolyncole  Would it make more sense if `#create!` returned true on a change and false if no change occurs?